make-util.js

```diff
@@ -19,7 +19,13 @@ var makeOptions = require('./make-options.json');
 // list of .NET culture names
 var cultureNames = [ 'cs', 'de', 'es', 'fr', 'it', 'ja', 'ko', 'pl', 'pt-BR', 'ru', 'tr', 'zh-Hans', 'zh-Hant' ];
 
-var allowedTypescriptVersions = ['2.3.4', '4.0.2'];
+// core dev-dependencies constants
+const constants = require('./dev-dependencies-constants');
+
+const MOCHA_TARGET_VERSION = constants.MOCHA_TARGET_VERSION;
+const TSC_MIN_VERSION = constants.TSC_MIN_VERSION;
+
+var allowedTypescriptVersions = [TSC_MIN_VERSION, '4.0.2'];
```

Comment:
```suggestion
const allowedTypescriptVersions = [TSC_MIN_VERSION, TSC_CURRENT_VERSION];
```

Could you please move `'4.0.2'` to consts?